stacks/its.js

```diff
@@ -0,0 +1,129 @@
+const { addBaseOptions, addOptionsToCommands, getWallet } = require('./utils');
+const { makeContractCall, PostConditionMode, AnchorMode, broadcastTransaction, Cl } = require('@stacks/transactions');
+
+const AXELAR_HUB_IDENTIFIER = 'hub';
+
+async function setTrustedAddress(privateKey, networkType, config, chain, args) {
+    const contracts = chain.contracts;
+    if (!contracts.InterchainTokenService?.address) {
+        throw new Error(`Contract InterchainTokenService not yet deployed`);
+    }
+    if (!contracts.InterchainTokenServiceImpl?.address) {
+        throw new Error(`Contract InterchainTokenServiceImpl not yet deployed`);
+    }
+
+    const trustedChains = parseTrustedChains(config.chains, args);
+
+    for (const trustedChain of trustedChains) {
+        printInfo(`Setting trusted address for ${trustedChain}`);
+
+        const itsAddress = contracts.InterchainTokenService.address.split('.');
+        const registerTransaction = await makeContractCall({
+            contractAddress: itsAddress[0],
+            contractName: itsAddress[1],
+            functionName: 'set-trusted-address',
+            functionArgs: [
+                Cl.address(contracts.InterchainTokenServiceImpl.address),
+                Cl.stringAscii(trustedChain),
+                Cl.stringAscii(AXELAR_HUB_IDENTIFIER),
+            ],
+            senderKey: privateKey,
+            network: networkType,
+            postConditionMode: PostConditionMode.Allow,
+            anchorMode: AnchorMode.Any,
+            fee: 10_000,
+        });
+        const result = await broadcastTransaction({
+            transaction: registerTransaction,
+            network: networkType,
+        });
+
+        printInfo(`Finished setting ${trustedChain} as trusted`, result.txid);
+
+        // Wait a bit before executing next transaction
+        await new Promise((resolve) => setTimeout(resolve, 5_000));
+    }
+}
+
+async function removeTrustedAddress(privateKey, networkType, config, chain, args) {
+    const contracts = chain.contracts;
+    if (!contracts.InterchainTokenService?.address) {
+        throw new Error(`Contract InterchainTokenService not yet deployed`);
+    }
+    if (!contracts.InterchainTokenServiceImpl?.address) {
+        throw new Error(`Contract InterchainTokenServiceImpl not yet deployed`);
+    }
+
+    const trustedChains = parseTrustedChains(config.chains, args);
+
+    for (const trustedChain of trustedChains) {
+        printInfo(`Removing trusted address for ${trustedChain}`);
+
+        const itsAddress = contracts.InterchainTokenService.address.split('.');
+        const registerTransaction = await makeContractCall({
+            contractAddress: itsAddress[0],
+            contractName: itsAddress[1],
+            functionName: 'remove-trusted-address',
+            functionArgs: [Cl.address(contracts.InterchainTokenServiceImpl.address), Cl.stringAscii(trustedChain)],
+            senderKey: privateKey,
```

Comment:
create a simpler helper around it perhaps that just takes the wallet, contractConfig (extracting name/address/networkType from it), function name and args and sets the other fields automatically, and also broadcasts the tx?